setup.cfg

```diff
@@ -1,3 +1,3 @@
 [flake8]
-max-line-length = 88
-ignore = E203, W503, E501, E231  # Black
+max-line-length = 2424
```

Comment:
**Issue**:  This line length does not match:

https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length

Reply:
Let’s try